daisy/persistence/mongodb_graph_provider.py

```diff
@@ -303,7 +317,7 @@ def read_edges(self, roi, nodes=None, attr_filter=None, read_attrs=None):
             # limit query to 1M node IDs (otherwise we might exceed the 16MB
             # BSON document size limit)
             length = len(node_ids)
-            query_size = 1000000
```

Comment:
don't remember why this was changed. Not sure if its needed

Reply:
I guess if you query both endpoints for the "or" case, half the "query size" takes up the same amount of space in the query. Since the point of the limit is to avoid exceeding the BSON document size limit, this makes sense.

Reply:
right that query size limit sounds very familiar